### DISCUSSION TITLE
Announcement: To source-build .NET 6.0.103, use dotnet/installer's v6.0.103-source-build tag

### DISCUSSION THREAD

**Discussion by lbussell** [dotnet/source-build#2789] [Announcements]
# To source-build 6.0.103, use dotnet/installer's v6.0.103-source-build tag

## Details
Due to an issue in [dotnet/source-build-reference-packages](https://github.com/dotnet/source-build-reference-packages), dotnet/installer's v6.0.103 tag introduces some non-deterministic package output. Maintainers and others wanting to build the 6.0.103 source-built SDK should use the [v6.0.103-source-build](https://github.com/dotnet/installer/releases/tag/v6.0.103-source-build) tag instead.

The commit that introduced the issue has been reverted, so it shouldn't cause issues in future releases.

## Action Items
Distro maintainers and others should use the [v6.0.103-source-build tag](https://github.com/dotnet/installer/releases/tag/v6.0.103-source-build) to build source-build.

## Related Issues

- https://github.com/dotnet/source-build-reference-packages/pull/349
- https://github.com/dotnet/source-build-reference-packages/pull/351

**Comment by omajid** [comment]
cc @alucryd @asbjornu @ayakael @cho-m @crummel @Livius90 @mirespace

**Reply by ayakael** [reply]
Many thanks! Does this bug affect `v6.0.201` tag?

**Reply by MichaelSimons** [reply]
Source-build is currently only supported in the 6.0.1xx channel.  See https://github.com/dotnet/source-build#support.

**Reply by asbjornu** [reply]
Thanks for the notification, @omajid. As described in #1836, I would love to automate the bump of the Homebrew formula. Could someone please engage with me in that issue so we can get that automation implemented?

**Reply by mirespace** [reply]
Thanks for the CC Omair!
